Yoda/test.cc

```diff
@@ -54,13 +59,23 @@ TEST(Yoda, WritesToFile) {
   typedef yoda::SingleFileAPI<Dictionary<YodaEntryToPersist>> PersistingAPI;
   PersistingAPI api("WritingToFileAPI", persistence_file_name);
 
-  api.Add(YodaEntryToPersist("one", 1));
-  api.Add(YodaEntryToPersist("two", 2));
+  api.Transaction([](PersistingAPI::T_DATA data) {
+    auto adder = Dictionary<YodaEntryToPersist>::Mutator(data);
+    adder.Add(YodaEntryToPersist("one", 1));
+    adder.Add(YodaEntryToPersist("two", 2));
+  }).Wait();;
   while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data.size()) {
     ;  // Spin lock.
   }
-
   EXPECT_EQ(yoda_golden_data, bricks::FileSystem::ReadFileAsString(persistence_file_name));
+
+  api.Transaction([](PersistingAPI::T_DATA data) {
+    Dictionary<YodaEntryToPersist>::Mutator(data).Delete("one");
+  }).Wait();
+  while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data_after_delete.size()) {
```

Comment:
Ah, see (went through emails sequentially). How about checking `> 0` instead? The exact value (and, automagically, its length) will be checked below.